features/0434-outofband/README.md

```diff
@@ -427,11 +427,11 @@ Knowledge is Good
 It seems inevitable that the length of some out-of-band message will be too long to produce a useable QR code. Techniques to avoid unusable QR codes have been presented above, including using attachment links for requests, minimizing the routing of the response and eliminating unnecessary whitespace in the JSON. However, at some point a _sender_ may need generate a very long URL. In that case, a URL shortener redirection should be implemented by the sender as follows:
 
 - The sender should generate and track a GUID for the out-of-band message URL.
-- The shortened version should be:
+- The shortened version should be of the form:
   - `https://example.com/ssi?id=5f0e3ffb-3f92-4648-9868-0d6f8889e6f3`
   - Note the replacement of the query parameter `oob` with `id` when using shortened URL.
-- On receipt of this form of message, the agent must do an HTTP GET to retrieve the associated, encoded out-of-band message.
-  - A sender may want to wait to generate the full invitation until the redirection event of the shortened URL to the full length form dynamic, so a single QR code can be used for distinct out of band messages.
+- On receipt of this form of message, the agent must do an HTTP GET to retrieve the associated out-of-band message. The agent should include an `Accept` header requesting the `application/json` MIME type, and the sender must include a `Content-Type` header specifying `application/json; charset=utf-8`. The sender must return the invitation in JSON format.
```

Comment:
as `must` is a keyword for rfcs can we make it all caps?

I really like the update.